README.md

```diff
@@ -252,6 +252,7 @@ your browser.
 
 ## 3. Social accounts application keys
 
+
```

Comment:
Kapitoly aj podkapitoly su vsade odsadene o dva riadky. Pre konveciu som to opravil.